### PR TITLE
Fix linting errors

### PR DESCRIPTION
**Pull request by robbevp**
@charvp merging #276 after #301 caused some linting errors. Fixed here

### CODE REVIEW

File: src/store/genres.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import Vue from "vue";
 import {
   index,
   create,
+  read,
```
Comment (charvp) [inline src/store/genres.js:5]: These changes are unrelated?

Comment (robbevp) [reply src/store/genres.js:5]: No, was also something picked up by eslint.

**Review by charvp — APPROVED** [review]
